admin_pages/maintenance/templates/ee_system_stati_page.template.php

```diff
@@ -29,7 +29,7 @@ function ee_recurse_into_array_for_display($data, $depth = 0, $td = true)
         $depth++;
         ksort($data, SORT_NATURAL | SORT_FLAG_CASE);
         if (EEH_Array::is_associative_array($data)) { ?>
-            <table class='ee-system-stati ee-system-stati-<?php echo $depth; ?>'>
+            <table class='ee-system-stati ee-system-stati-<?php echo sanitize_html_class($depth); ?>'>
```

Comment:
or

```suggestion
            <table class='ee-system-stati ee-system-stati-<?php echo absint($depth); ?>'>
```

since this is an integer? (see line 29 just above this)

`sanitize_html_class()` gets into using regex which is totally unnecessary here